### PR TITLE
added Installation Guide for C++ on MAC OS

### PR DESCRIPTION
**Pull request by Nidakhan786**
Added README for installation guide for C++ on MAC OS

### CODE REVIEW

File: guides/installation_guides/c++/mac/README.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
+<b1> <b>Getting The Software</b> </b1> <br> <br>
+>There are many IDE's for MAC OS but here's a quick guide to install one of them i.e <b> CODEBLOCKS>. <br>
```
Comment (arnavb) [inline guides/installation_guides/c++/mac/README.md]: Why is everything in a blockquote? Don't overuse formatting.

File: guides/installation_guides/c++/mac/README.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
+<b1> <b>Getting The Software</b> </b1> <br> <br>
```
Comment (arnavb) [inline guides/installation_guides/c++/mac/README.md]: What  is the `<b1>` tag? Also, for bolding, you can put the text inside two stars: `**text**`.

File: guides/installation_guides/c++/mac/README.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+<img src="https://github.com/Nidakhan786/hello-world/blob/master/images/6.png"> <br> <br>
+>Step 6: Once the installation resumes, and if you don’t have Xcode, you’ll be prompted by this message. Click on Get Xcode.<br><br>
+<img src="https://github.com/Nidakhan786/hello-world/blob/master/images/7.png"> <br> <br>
+>This will take you to the Mac AppStore. Here, after entering your Apple ID and Password you’ll be able to install Xcode for free. Depending on your Internet Speed, it could take some time to download and Install the 4.5gb application.<br><br>
+<img src="https://github.com/Nidakhan786/hello-world/blob/master/images/8.png"> <br> <br>
+>Step 7: Once Xcode is installed, head back over to Code Blocks and start a new project. Opt for a console application from the different templates available and select C as the language.<br><br>
+<img src="https://github.com/Nidakhan786/hello-world/blob/master/images/9.png"> <br> <br>
+>Step 8: Select the path where you want to save your file.<br><br>
+<img src="https://github.com/Nidakhan786/hello-world/blob/master/images/10.png"> <br> <br>
+><h2 align ="center"><b> Start coding!! :D </></h2>
```
Comment (arnavb) [inline guides/installation_guides/c++/mac/README.md]: Don't forget the cosmos tagline!

File: guides/installation_guides/c++/mac/README.md (hunk 1 above)
Comment (Nidakhan786) [reply guides/installation_guides/c++/mac/README.md]: okayy , i have updated it . you can check it now..
Thank you!

File: code/unclassified/src/leap_year/Leap_Year.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+ 
+public class Leap_Year {
```
Comment (arnavb) [inline code/unclassified/src/leap_year/Leap_Year.java]: LeapYear, not Leap_Year.

File: code/unclassified/src/leap_year/Leap_Year.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import java.util.*;
```
Comment (arnavb) [inline code/unclassified/src/leap_year/Leap_Year.java]: file should be called leap_year.java.

File: code/unclassified/src/leap_year/Leap_Year.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+			}
+		}
+	}
+}
+ 
+class MainClass {
+	
+	public static void main(String str[]) {
+		
+		Leap_Year obj = new Leap_Year();
```
Comment (arnavb) [inline code/unclassified/src/leap_year/Leap_Year.java]: Move all console related stuff to main, and instead use return values in the other class.

File: code/unclassified/src/leap_year/Leap_Year.java [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+		System.out.println("\nEnter the first year : ");
+		year1 = Integer.parseInt(sc.nextLine());
+		
+		System.out.println("Enter the second  : ");
+		year2 = Integer.parseInt(sc.nextLine());
+	}
+	// Printing no of leap year between two years
+	void checkLeapYear() {
+
+		System.out.println();
```
Comment (arnavb) [inline code/unclassified/src/leap_year/Leap_Year.java]: This class should _not_ print to the console.

**Pull request closed without merging**